moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -618,8 +618,17 @@ bool ServoCalcs::internalServoUpdate(Eigen::ArrayXd& delta_theta,
   // Enforce SRDF position limits, might halt if needed, set prev_vel to 0
   const auto joints_to_halt = enforcePositionLimits(next_joint_state_, servo_params_.joint_limit_margin,
                                                     joint_model_group_, *node_->get_clock());
+
   if (!joints_to_halt.empty())
   {
+    std::ostringstream joints_names;
+    std::transform(joints_to_halt.cbegin(), std::prev(joints_to_halt.cend()),
+                   std::ostream_iterator<std::string>(joints_names, ", "),
+                   [](const auto& joint) { return joint->getName(); });
+    joints_names << joints_to_halt.back()->getName();
+    RCLCPP_WARN_STREAM_THROTTLE(LOGGER, *node_->get_clock(), ROS_LOG_THROTTLE_PERIOD,
+                                joints_names.str() << " close to a position limit. Halting.");
```

Comment:
right now this will print something like

```
joint1 joint2 joint3 close to a position limit. Halting.
```

Consider formatting it a bit more clearly , for example

```
Joints 'joint1' 'joint2' 'joint3' close to a position limit. Halting.
```